tests/diagnostics/charged_particle_radiography/test_synthetic_radiography.py

```diff
@@ -433,6 +436,17 @@ def test_create_particles() -> None:
     # Test specifying particle
     sim.create_particles(1e3, 15 * u.MeV, particle="e-", random_seed=42)
 
+    # Test specifying direction
+    src_vdir = np.array([0.1, 1, 0])
+    src_vdir /= np.linalg.norm(src_vdir)
+    sim.create_particles(
+        1e3, 15 * u.MeV, particle="p+", random_seed=42, source_vdir=src_vdir
+    )
+    # Assert particle velocities are actually in that direction
+    vdir = np.mean(sim.v, axis=0)
+    vdir /= np.linalg.norm(vdir)
```

Comment:
```suggestion
    vdir = np.mean(sim.v, axis=0) / np.linalg.norm(vdir)
```
Would it make sense to combine these lines?

Reply:
Sure!

Reply:
I don't think it will work, because vdir in norm is then not defined yet?